app/webpacker/styles/lead_providers.scss

```diff
@@ -0,0 +1,537 @@
+.index-group-lead-provider {
+  background: govuk-colour("light-grey");
+  padding: 1.5em;
+  margin-bottom: 1.5em;
+}
+
+
+.app-list--definition {
+  dt {
+    @include govuk-typography-weight-bold;
+  }
+
+  dd + dt {
+    margin-top: govuk-spacing(4);
+  }
+
+  dd {
+    margin-left: 0;
+  }
+
+  dd + dd {
+    margin-top: govuk-spacing(2);
+  }
+}
+
+.gem-c-contents-list__list, .gem-c-contents-list__nested-list {
+  font-size: 16px;
+  line-height: 1.25;
+}
+
+.gem-c-contents-list__list-item--dashed {
+  position: relative;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.gem-c-contents-list__list-item {
+  padding-top: 7.5px;
+  line-height: 1.3;
+  list-style-type: none;
+}
+
+.gem-c-contents-list__list-item--dashed:before {
+  content: "—";
+  position: absolute;
+  left: 0;
+  width: 20px;
+  overflow: hidden;
+}
+
+a.signout {
+  color: #FFF;
+  font-weight: 700;
+  position: absolute;
+  right: calc(50% - 475px);
+  top: 15px;
+}
+
+
+.search-filter {
+  padding: 15px 10px 10px;
+  background: #f5f5f5;
+  margin-bottom: 20px;
+}
+.filter-group {
+  position: relative;
+}
+.clear-filters {
+  font-family: "GDS Transport",arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  line-height: 1.1428571429;
+  position: absolute;
+  right: 0px;
+  top: 5px;
+  text-align: right;
+}
+.search-filter .govuk-checkboxes--small .govuk-checkboxes__label:before {
+  background: #FFF;
+}
+.search-filter .govuk-checkboxes--small .govuk-checkboxes__label {
+  font-size: 17px;
```

Comment:
I think font sizes and margins should be handled with govuk design system classes.

I feel REALLY strongly about it.

Reply:
Deleted anything not used